tests/test_authenticators.py

```diff
@@ -78,6 +78,14 @@ def assertCredMismatch(self, cred, uid, gid):
         request = {'creds': {'uid': uid, 'gid': gid}, 'client_id': 'tests'}
         self.assertFalse(cred.handle(request))
 
+    def assertHeaderMatch(self, header, key, value, client_id):
+        request = {'headers': {key: value}, 'client_id': client_id}
+        self.assertTrue(header.handle(request))
```

Comment:
assertTrue and assertFalse only perform checks ```if expr``` and ```if not expr```. We should rather test for ``assertIs(expr, True)``.